deploy/device-operator.yaml

```diff
@@ -515,8 +514,8 @@ spec:
           securityContext:
             privileged: true
             allowPrivilegeEscalation: true
-          image: openebs/device-driver:ci
-          imagePullPolicy: IfNotPresent
+          image: akhilerm/device-driver:ci
+          imagePullPolicy: Always
```

Comment:
why always?

Reply:
Yes, its debug code, Checked in accidentally.

Reply:
Done @pawanpraka1